CMakeLists.txt

```diff
@@ -59,7 +61,9 @@ find_package(OpenMP COMPONENTS Fortran)
 find_package(MPI REQUIRED COMPONENTS Fortran)
 find_package(NetCDF REQUIRED COMPONENTS Fortran C)
 find_package(PnetCDF REQUIRED COMPONENTS Fortran)
-find_package(PIO REQUIRED COMPONENTS Fortran C)
+if(NOT MPAS_USE_SMIOL)
```

Comment:
This doesn't seem to work. When I build with `-DMPAS_USE_SMIOL=1`  the cmake output still has:
```
-- Found PIO: /glade/work/epicufsrt/contrib/spack-stack/derecho/spack-stack-1.6.0/envs/unified-env/install/gcc/12.2.0/parallelio-2.5.10-ysh2ljm  found components: Fortran C 
-- FindPIO:
--   - PIO_PREFIX [/glade/work/epicufsrt/contrib/spack-stack/derecho/spack-stack-1.6.0/envs/unified-env/install/gcc/12.2.0/parallelio-2.5.10-ysh2ljm]
--   - PIO Components Found: Fortran;C;SHARED
```
as well as
```
-- The following REQUIRED packages have been found:
...
 * PIO
```